huaweicloud/services/apig/resource_huaweicloud_apig_instance.go

```diff
@@ -594,12 +594,15 @@ func updateInstanceTags(client *golangsdk.ServiceClient, d *schema.ResourceData)
 
 func resourceInstanceUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	cfg := meta.(*config.Config)
-	client, err := cfg.ApigV2Client(cfg.GetRegion(d))
+	region := cfg.GetRegion(d)
+	client, err := cfg.ApigV2Client(region)
 	if err != nil {
 		return diag.Errorf("error creating APIG v2 client: %s", err)
 	}
 
-	// Update egress access
+	instanceId := d.Id()
+
+	// Update egress acces
```

Comment:
```suggestion
	// Update egress access
```

Reply:
ok, this was mistakenly deleted, thank you for pointing out